src/index.ts

```diff
@@ -1,12 +1,18 @@
 // eslint-disable-next-line import/no-default-export
 export { default } from './utils/OneUI';
+export {
+    HighlighterCore,
+    HighlighterCoreOptions,
+    HighlighterCoreAccuracy,
+} from './utils/HighlighterCore';
```

Comment:
do we really need to export it as a part of OneUI public functionality?

Reply:
Why not? :) I think the method can be useful when we need to highlight or find some text, but don't need the React component for this, on some low level. Actually I think the place for 
 `HighlighterCore` is a separate library.

Reply:
Well, there is this principle "Expose as little as you can", otherwise you have to support and maintain what you have exposed. But, in our case I guess it's ok.


> Actually I think the place for HighlighterCore is a separate library.
Might be then it's better to move it to  `/src/packages`?

Reply:
I would agree with @antipin, we can always expose it once we really need it to be exposed? Unless of course you already foresee that to be the case, I can't tell

Reply:
Cool, I will move it to`/src/packages`.
The idea was to use some `HighlighterCore` methods for Reach.
For instance, `StringHighlighter` applies some terms ['JavaScript', 'Göttingen', 'php developer']. Every time when `StringHighlighter` component is run it does some normalization for that terms: some sorting, deleting diacritics, escaping. So if that component mounted 20 times it will do the normalization for the same terms 20 times :(.
For this reason, I added support `RegExp` for `StringHighlighter`. `HighlighterCore` exposes methods that allow to create `RegExp`. Thus we can create `RegExp` outside `StringHighlighter` and then pass that `RegExp` for every `StringHighlighter` instance. It is a kind of performance improvement.